src/main/java/org/opentripplanner/transit/raptor/api/transit/RaptorTransfer.java

```diff
@@ -22,6 +22,8 @@
      */
     int stop();
 
+    int cost();
```

Comment:
This need java doc, we should also use `generalizedCost` as method name. Doc:
 - Unit is cost points 1 centi-second of riding-transit. 
 - Used for/as part of
 - How to implement it - Avoid calculations, preferably cash it.